extensions/disruptor/runtime/pom.xml

```diff
@@ -87,6 +92,8 @@
                             <version>${quarkus.version}</version>
                         </path>
                     </annotationProcessorPaths>
+                    <source>9</source>
```

Comment:
Should this not be 11 instead of 9 if we go higher than 8

Reply:
looks like some IDE intentions applied, fixing it